src/System.Net.HttpListener/tests/System.Net.HttpListener.Tests.csproj

```diff
@@ -10,26 +10,26 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='netstandard-Release|AnyCPU'" />
   <ItemGroup>
     <Compile Include="GetContextHelper.cs" />
+    <Compile Include="HttpListenerFactory.cs" />
```

Comment:
You're right. These two are just helpers: I reordered this so it's easier to comment out tests.